test/alternator/test_batch.py

```diff
@@ -274,6 +274,27 @@ def test_batch_get_item_hash(test_table_s):
     got_items = reply['Responses'][test_table_s.name]
     assert multiset(got_items) == multiset(items)
 
+# A single GetBatchItem batch can ask for items from multiple tables, let's
```

Comment:
I'm not sure if it's needed (would be happy to learn what's the procedure in such cases myself), but this commit can be squeezed with the commit that introduced authz to BatchGetItem.

Reply:
This test wasn't about the authorization of GetBatchItem... I just realized, while adding authorization for GetBatchItem, that I never wrote a test for GetBatchItem which reads from multiple tables, so I thought it would be a good idea to add one. I could have sent it as a completely separate PR but thought I would just stick it as a separate patch.